src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const DAY_RENT = 40;
+  const BigDiscount = 50;
+  const SmallDiscount = 20;
+
+  if (days < 3) {
+    return days * DAY_RENT;
+  } else if (days >= 3 && days < 7) {
+    return days * DAY_RENT - SmallDiscount;
+  }
```

Comment:
The `else if` is unnecessary since the previous `if` block contains a `return` statement. Removing the `else` will simplify the code structure and adhere to checklist items 4 and 5.